controllers/common_test.go

```diff
@@ -292,6 +292,12 @@ func kind(kind string) schema.GroupKind {
 // 			Result:       nil,
 // 		},
 // 		{
+// 			Name:         "Test: ensureSubscription - Cluster Proxy Addon",
```

Comment:
Did you mean to leave this commented out?

Reply:
Yes, that entire file is commented out so I added the entry without changing that